Code_Gantt.js

```diff
@@ -5,21 +5,23 @@
 function getEntireGanttSheet(width, height) {
```

Comment:
Let's give the function JSDoc commenting. See #13 for an example, as well as #10.